hw/ip/usbdev/dv/usbdev_sim_cfg.hjson

```diff
@@ -165,6 +165,11 @@
       // Short sequence, many possible endpoint configurations.
       reseed: 200
     }
+    {
+      name: usbdev_fifo_levels
+      uvm_test_seq: usbdev_fifo_levels_vseq
+      reseed: 500
```

Comment:
Is 500 really the count we need here?

Reply:
Well, it's relying upon randomization to hit coverage bins, and there were 192 possibilities. Each sequence itself is very short, but I've looked again at the coverage bins and noted that it was never the intent to exercise the RX FIFO with a level of 1 or the Av FIFOs (from which buffers are pulled by the DUT) with a level of 'depth - 1' since these are not corner cases,  so we're down to 81 possibilities. I've dropped the seed count to 160 accordingly.